### ISSUE TITLE
Kill preprocessor

### ISSUE THREAD

**Issue by simonsmith** [suitcss/suit#162]
Most of the features in there have native browser support already, and it's very easy to hook up `postcss-cli` otherwise. 

Either that or we could create a plugin pack, [similar to this](https://github.com/alanev/postcss-pack-alanev/blob/master/index.js).

**Comment by giuseppeg** [comment]
+1 for creating the preset

**Comment by mlnmln** [comment]
Agreed. From a consumers' perspective it's more transparent to depend on a preset. 

Edit: We should either create a plugin pack or consider just adopting postcss-preset-env https://github.com/csstools/postcss-preset-env. Which would stay true to the "future facing CSS" paradigm, would'n it?
